packages/backend/models/unified/note.ts

```diff
@@ -36,14 +38,14 @@ export function unifyNote(note: any, tpId: TP_ID): UnifiedNote {
         additional: {},
         associations: {
             ...(tpId === TP_ID.pipedrive && {
-                personId: note.person_id,
-                organizationId: note.org_id,
+                contactId: note.person_id,
+                companyId: note.org_id,
                 leadId: note.lead_id,
                 dealId: note.deal_id,
             }),
-            ...(tpId === TP_ID.hubspot && {
-                deal_id: '',
-            }),
+            // ...(tpId === TP_ID.hubspot && {
```

Comment:
@Hvinder Is this commented out for a reason?

Reply:
yeah, we don't need that as we don't get the associated `deal_id` in the response